src/role_scout.js

```diff
@@ -17,17 +17,44 @@ roles.scout.settings = {
  * @param {object} creep
```

Comment:
`* @param {Creep} creep`

Reply:
I would rather do it in another seperate pr, as:
![image](https://user-images.githubusercontent.com/17455337/189367919-15e5db91-961d-4d1b-a704-70edc96d3a26.png)

Reply:
sure, fine with that

Reply:
i'll ping you on that

Reply:
> i'll ping you on that

ping me when your doc fix pr in. or would bring large amout of conflicts.